test/Microsoft.AspNetCore.Mvc.FunctionalTests/HttpResponseMessageExceptions.cs

```diff
@@ -7,7 +7,7 @@
 using Xunit;
 using Xunit.Sdk;
 
-namespace FiltersWebSite
+namespace Microsoft.AspNetCore.Mvc.FunctionalTests
```

Comment:
It's on a test project. I just moved it out of filter websites cause I killed the Configuration project (that was bringin in the Xunit dependency to the website)